### DISCUSSION TITLE
Updating an outdated cask's version

### DISCUSSION THREAD

**Discussion by giacomocavalieri** [Homebrew/discussions#4703] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 98f1be77f93a0d23fbc68629d0854522e9f0c1c3
Last commit: 2 days ago
Core tap JSON: 09 Aug 11:30 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hello! I noticed the Pokemon-tcg-live formula is outdated, the current release is version `1.7.0.195880` and I was wondering what would it take for me to update brew's version. By looking at the previous commits it should be as simple as making a PR with the new version but I'm not sure if extra steps are required.
My question is, what is the common procedure to update the version of a cask's formula like this one? Thank you :)

**Comment by ZhongRuoyu** [comment, marked as answer]
Please see https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#submit-a-new-version-of-an-existing-cask:

> Use `brew bump-cask-pr` to do everything (i.e. forking, committing, pushing) with a single command. Run `brew bump-cask-pr --help` to learn more.

N.b. you may need to run `brew tap homebrew/cask` before doing that.

**Reply by giacomocavalieri** [reply]
Thank you for the answer!
